Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.yml

```diff
@@ -0,0 +1,169 @@
+  required: false
+  type: 19
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: Supports CSV values.
+  display: Tags
+  name: feedTags
+  required: false
+  type: 0
+- display: The Crowdstrike api client_secret
+  hidden: false
+  name: client_secret
+  required: true
+  type: 4
+- additionalinfo: List divided by commas. for example - Aerospace,Academic
+  display: Filter by actor's target industries.
+  hidden: false
+  name: target_industries
+  required: false
+  type: 0
+- additionalinfo: List divided by commas. for example - united states,canada
+  display: Filter by actor's target countries.
+  hidden: false
+  name: target_countries
+  required: false
+  type: 0
+- additionalinfo: Link to documentation how to filter - https://falcon.crowdstrike.com/login/?next=%2Fsupport%2Fdocumentation%2F45%2Ffalcon-query-language-fql%3F
+  display: Filter by custom filter, If the user uses custom_filter other filters will
+    not work
+  hidden: false
+  name: custom_filter
+  required: false
+  type: 0
+description: CrowdStrike intelligence team are tracking the activities of threat actor
+  groups and  advanced persistent threats (APTs) to understand as much as possible
+  about their known aliases, targets, methods, and more.
+display: Crowdstrike Falcon Intel Feed
+name: Crowdstrike Falcon Intel Feed
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The maximum number of indicators to return. Default is 10.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: The index of the first indicator to fetch.
+      isArray: false
+      name: offset
+      required: false
+      secret: false
+    - default: false
+      description: Filter by actor's target industries, List divided by commas.
+      isArray: false
+      name: target_industries
+      required: false
+      secret: false
+    - default: false
+      description: Filter by actor's target countries, List divided by commas.
+      isArray: false
+      name: target_countries
+      required: false
+      secret: false
+    - default: false
+      description: Filter by custom filter, If the user uses custom_filter other filters
+        will not work
+      isArray: false
+      name: custom_filter
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets indicators from Crowdstrike Falcon Intel Feed.
+    execution: false
+    name: crowdstrike-falcon-intel-get-indicators
```

Comment:
The command has no outputs? please add them and then re-create the readme

Reply:
This is a command only for testing and therefore has no outputs